libs/dh/admin/data-access-api/src/lib/dh-admin-user-roles-management-data-access-api.store.ts

```diff
@@ -67,14 +67,15 @@ export class DhAdminUserRolesManagementDataAccessApiStore
   roles$ = this.select((state) => state.roles);
 
   rolesFiltered$ = this.select(this.roles$, this.filterModel$, (roles, filter) =>
-    roles.filter(
-      (role) =>
+    roles.filter((role) => {
```

Comment:
question: Why no implicit return?

Reply:
I forgot to undo - needed debug statement. :P